nipype/interfaces/afni/base.py

```diff
@@ -157,6 +157,11 @@ def __init__(self, **inputs):
         else:
             self._output_update()
 
+        # Update num threads estimate from OMP_NUM_THREADS env var
+        # Default to 1 if not set
+        import os
+        self.num_threads = int(os.getenv('OMP_NUM_THREADS', 1))
```

Comment:
Shouldn't we do this other way around? If user sets num_threads to say 4 we would set the OMP_NUM_THREADS variable to 4? I think this is how it's done for ants: https://github.com/nipy/nipype/blob/master/nipype/interfaces/ants/base.py#L52